JPS_Ontology/ontology/ontosensorloggermobileapp/OntoSensorLoggerMobileApp.csv

```diff
@@ -0,0 +1,25 @@
+Source,Type,Target,Relation,Domain,Range,Quantifier,Comment,Defined By,Label
+OntoSensorLoggerMobileApp,TBox,http://www.theworldavatar.com/ontology/ontosensorloggermobileapp/ontoSensorLoggerMobileApp.owl,https://www.w3.org/2007/05/powder-s#hasIRI,,,,,,
+OntoSensorLoggerMobileApp,TBox,1,http://www.w3.org/2002/07/owl#versionInfo,,,,,,
+OntoSensorLoggerMobileApp,TBox,The OntoSensorLoggerMobileApp ontology (OntoSensorLoggerMobileApp) is an ontology developed to represent the detailed information and spatial relationships,http://www.w3.org/2000/01/rdf-schema#comment,,,,,,
+OntoSensorLoggerMobileApp,TBox,,http://www.w3.org/2002/07/owl#imports,,,,,,
+Vector,Class,,,,,,A quantity that has magnitude and direction.,https://www.theworldavatar.com/kg/ontosensorloggermobileapp/,Vector
+MagneticFluxDensityVector,Class,Vector,IS-A,,,,A vector quantity that represents the strength and direction of a magnetic field.,https://www.theworldavatar.com/kg/ontosensorloggermobileapp/,MagneticFluxDensityVector
+GravityVector,Class,Vector,IS-A,,,,A vector quantity that represents the direction and magnitude of the force of gravity.,https://www.theworldavatar.com/kg/ontosensorloggermobileapp/,GravityVector
```

Comment:
Force is not the same as acceleration, so I would recommend replacing "force of gravity" with "gravitational acceleration".